lib/statemanager.js

```diff
@@ -446,7 +446,17 @@ StateManager.prototype.sign = function(address, dataToSign) {
   var msg = to.buffer(dataToSign, "hex");
   var msgHash = utils.hashPersonalMessage(msg);
   var sgn = utils.ecsign(msgHash, Buffer.from(secretKey));
-  return utils.toRpcSig(sgn.v, sgn.r, sgn.s);
+  // ethereumjs-utils changed the behavior of toRpcSig so that the `v` value
+  // output by `toRpcSig` is always `sgn.v - 27` (basically `0` or `1`). In
+  // oder to avoid a breaking change in Ganache at this time we are calculating
+  // a chain iID that will always a) validiate the v and b) generate an output v
+  // of `0` or `1` like it used to.
+  // Additionally, the output of 27/28 vs 0/1 is uncertain and hasn't yet been
+  // decided upon (geth apparently using both?).
+  // See: https://github.com/OpenZeppelin/openzeppelin-solidity/pull/1622
```

Comment:
We have an update on this.  https://github.com/ethereum/go-ethereum/issues/19751#issuecomment-504900739